### ISSUE TITLE
XSO Parser no longer displays the hard cap of an instance

### ISSUE THREAD

**Issue by SalbugVR** [nnaaa-vr/XSOverlay-VRChat-Parser#15]
Ever since the recent security updates, XSO log parser displays instance numbers as [x/0]

![XSOverlay_VRChat_Parser_KncGkjRy5i](https://user-images.githubusercontent.com/24326270/184521952-2cb71dca-283b-4200-891e-7b3120039b9c.png)

**Comment by nnaaa-vr** [comment]
Yeah, seems the max isn't printed to the log anymore. I noticed, but haven't had the time yet to look further just yet. I'll either find another way, get it re-added, or remove the cap tracking when I get to it.
